app/jobs/prepare_move_notifications_job.rb

```diff
@@ -5,7 +5,8 @@ class PrepareMoveNotificationsJob < ApplicationJob
   include QueueDeterminer
 
   def perform(topic_id:, action_name:, queue_as:, send_webhooks: true, send_emails: true, only_supplier_id: nil)
-    move = Move.find(topic_id)
+    topic = find_topic(topic_id)
```

Comment:
Now that this class's focus is the `topic` rather than the `move`, it feels misnamed... how about a new base class `PrepareNotificationsJob` from which `PrepareMoveNotificationsJob` and `PreparePersonEscortRecordNotificationsJob` can inherit?

good work not changing the method signature, as that will break on deployment if there are existing notifications for processing 👍